src/js/cookie-message.js

```diff
@@ -165,7 +171,13 @@ class CookieMessage {
 	 */
 	removeCookieMessage() {
 		this.dispatchEvent('oCookieMessage.close');
-		this.cookieMessageElement.parentNode.removeChild(this.cookieMessageElement);
+
+		try {
+			this.cookieMessageElement.parentNode.removeChild(this.cookieMessageElement);
```

Comment:
👍 maybe we should dispatch the close event after attempting to remove the node, so it's only fired once?

Reply:
Interesting idea...

Reply:
My only reservation is that this is only needed because of the way the test fakes displaying the page without reloading. Wrapping the node removal in a try/catch block adds robustness, but I'm not sure that changing the event behaviour gives any benefit?